examples/tree_3d_dgsem/elixir_navierstokes_taylor_green_vortex.jl

```diff
@@ -16,7 +16,11 @@ equations_parabolic = CompressibleNavierStokesDiffusion3D(equations, mu=mu(),
 """
     initial_condition_taylor_green_vortex(x, t, equations::CompressibleEulerEquations3D)
 
-The classical inviscid Taylor-Green vortex.
+The classical inviscid Taylor-Green vortex, as found for instance in
```

Comment:
You are still running a viscous test case, not an inviscid one, aren't you?

Reply:
Yes, this is probably a copy-paste error that originates from 

https://github.com/trixi-framework/Trixi.jl/blob/6c97c48e53feb9fe372dc020cbd5f3e1e8fef458/examples/tree_3d_dgsem/elixir_euler_taylor_green_vortex.jl#L10-L15

and found its way into 

https://github.com/trixi-framework/Trixi.jl/blob/6c97c48e53feb9fe372dc020cbd5f3e1e8fef458/examples/tree_3d_dgsem/elixir_navierstokes_taylor_green_vortex.jl#L16-L21